master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,48 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``2.0.2-dev66`` ( ``2019-02-19`` )
```

Comment:
please remove this chunk from the commit. This is autogenerated code.

Reply:
removed